src/components/intern-app.html

```diff
@@ -1,8 +1,8 @@
 <link rel="import" href="~@banno/polymer/polymer-element.html">
-<link rel="import" href="./user-profile.html">
+<link rel="import" href="~@banno/polymer/lib/elements/dom-repeat.html">
```

Comment:
don't need this import here. only need to import it if you're using it in the file in which you're importing.